pkg/catalog/catalog.go

```diff
@@ -1711,6 +1717,99 @@ func (c *Catalog) PrepareExpiredCommits(ctx context.Context, repositoryID string
 	return c.Store.SaveGarbageCollectionCommits(ctx, repository, previousRunID)
 }
 
+func createParquetFile(entries []DBEntry) (string, error) {
```

Comment:
Seems like a very straightforward parquet file writer, can we extract it to somewhere common?